htdocs/reports/reports_testhistory.php

```diff
@@ -2079,13 +2106,70 @@ function change_to_bold() {
 	}
 }
 ?>
-<div>
+<div id="signature">
 Signed by:
 <label>
 	<?php $user = get_user_by_id($signature);
 	echo $user->actualName; ?></label>
 </div>
 
+<div id="print_div" class="hidden"></div> <!--Content when the 1 test per page checkmark is checked is moved here-->
+
+<script>
+		const reportTable = "#patient_table";
+		const copiedElements = ["#logo", "#lab_header", "#report_header", "#patient_info_header"]
+		const printDiv = '#print_div';
+		const setupPrint = () => {

```

Comment:
Have you tried this in the bundled Firefox (which I think is version ...6? or something very old?)

Unfortunately I think [arrow functions aren't supported that far back](https://caniuse.com/arrow-functions), you might want to consider replacing these with eg. `function setupPrint() { }` like the bad old days.

Reply:
Good point, I was actually thinking that es6 support probably wouldn't be feature complete I'll revert that to a function block.